tools/monorepo-utils/src/code-freeze/commands/branch/index.ts

```diff
@@ -53,7 +55,7 @@ export const branchCommand = new Command( 'branch' )
 		'Branch to create the release branch from. Default: trunk',
 		'trunk'
 	)
-	.action( async ( options: Options ) => {
```

Comment:
All the extra Options types I deleted, we don't need them because the `@commander-js/extra-typings` library magically generates the types inferred when you call `.option` or `.argument`. It's pretty neat